codegen-server-test/model/simple.smithy

```diff
@@ -41,20 +41,20 @@ resource Service {
 @http(method: "PUT", uri: "/service/{id}")
 @documentation("Service register operation")
 operation RegisterService {
-    input: RegisterServiceInput,
-    output: RegisterServiceOutput,
+    input: RegisterServiceInputRequest,
```

Comment:
I think the old naming is better. An operation's inputs and outputs are _not_ requests and responses, respectively, but rather model types that _get converted into_ requests and responses by the protocol implementation. It is up to the protocol implementation to decide how to translate operation input and output model types into requests and responses.